web/packages/teleport/src/Resources/ResourceCard.tsx

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { Box, ButtonBorder, Flex, Label, Text } from 'design';
+
+import { CheckboxInput } from 'design/Checkbox';
+import { ResourceIcon, ResourceIconName } from 'design/ResourceIcon';
+import {
+  ApplicationsIcon,
+  DatabasesIcon,
+  DesktopsIcon,
+  KubernetesIcon,
+  ServersIcon,
+} from 'design/SVGIcon';
+
+import { AgentKind, UnifiedResourceKind } from 'teleport/services/agents';
+
+const SingleLineBox = styled(Box)`
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+type Props = {
+  resource: AgentKind;
+};
+export const ResourceCard = ({ resource }: Props) => {
+  const name = agentName(resource);
+  const resIcon = agentIconName(resource);
+  const ResTypeIcon = agentTypeIcon(resource.kind);
```

Comment:
```suggestion
  const resTypeIcon = agentTypeIcon(resource.kind);
```

Any particular reason this one is capitalized?

Reply:
Yes. In React, every component that will be rendered in JSX, needs to be capitalized; otherwise, it will be rendered as a plain HTML or SVG tag. I guess this is for compatibility with future HTML standards, as well as Web Components.